test/integration/helpers.sh

```diff
@@ -171,19 +180,18 @@ function start_sim() {
         else
             tpm2_sim_port=$TPM2_SIMPORT
         fi
-        tpm2_mssim_tcti_expected_port=$((tpm2_sim_port + 1))
+        tpm2_sim_cmd_port=$((tpm2_sim_port + 1))
         echo "Attempting to start simulator on port: $tpm2_sim_port"
         $TPM2_SIM -port $tpm2_sim_port &
         tpm2_sim_pid=$!
         sleep 1
 
-        # Do not rely on whether netstat is present or not and directly fetch
-        # data in relevent /proc file
-        tpm2_sim_port_inode="$(awk -v port="$(printf ':%04X' "$tpm2_sim_port")" '$2 ~ port { print $10 }' /proc/net/tcp)"
-        tpm2_mssim_tcti_expected_port_inode="$(awk -v port="$(printf ':%04X' "$tpm2_mssim_tcti_expected_port")" '$2 ~ port { print $10 }' /proc/net/tcp)"
+        ${sock_tool} 2>/dev/null | grep ${TPM2_SIM} | grep ${tpm2_sim_port}
+        tpm2_sim_port_rc=$?
+        ${sock_tool} 2>/dev/null | grep ${TPM2_SIM} | grep ${tpm2_sim_cmd_port}
```

Comment:
Same problem here, that only checks whether any simulator has managed to bind to the port, not if it is the one we started.